src/ConvertInputFormat/main.cpp

```diff
@@ -109,6 +109,121 @@ bool checkVersionMatch(json const &epJSON)
     return true;
 }
 
+bool checkForUnsupportedObjects(json const &epJSON, bool convertHVACTemplate)
```

Comment:
New function to check for unsupported objects in format converter instead of relying on `parse_idf`.